dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/api/imports/ImportDDIServiceBeanTest.java

```diff
@@ -0,0 +1,637 @@
+                createPrimitiveFieldDTO(DatasetFieldConstant.dataCollectionSituation, "collecting data situation characteristics"));
+        
+        assertThat(socialBlock.getField(DatasetFieldConstant.actionsToMinimizeLoss)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.actionsToMinimizeLoss, "actions to minimize loss"));
+
+        assertThat(socialBlock.getField(DatasetFieldConstant.controlOperations)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.controlOperations, "control operation text"));
+
+        assertThat(socialBlock.getField(DatasetFieldConstant.weighting)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.weighting, "PostStratification, MixedPostStratificationDesign"));
+        
+        assertThat(socialBlock.getField(DatasetFieldConstant.cleaningOperations)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.cleaningOperations, "operations cleaning text"));
+
+        assertThat(socialBlock.getField(DatasetFieldConstant.socialScienceNotes)).isEqualTo(
+                createCompoundFieldDTO(DatasetFieldConstant.socialScienceNotes, 
+                        createPrimitiveFieldDTO(DatasetFieldConstant.socialScienceNotesSubject, "notes subject (social)"),
+                        createPrimitiveFieldDTO(DatasetFieldConstant.socialScienceNotesType, "type of notes (social)"),
+                        createPrimitiveFieldDTO(DatasetFieldConstant.socialScienceNotesText, "notes text (social)")));
+
+        assertThat(socialBlock.getField(DatasetFieldConstant.responseRate)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.responseRate, "99.9"));
+        
+        assertThat(socialBlock.getField(DatasetFieldConstant.samplingErrorEstimates)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.samplingErrorEstimates, "11"));
+        
+        assertThat(socialBlock.getField(DatasetFieldConstant.otherDataAppraisal)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.otherDataAppraisal, "appraisal method text"));
+
+        assertThat(citationBlock.getField(DatasetFieldConstant.notesText)).isEqualTo(
+                createPrimitiveFieldDTO(DatasetFieldConstant.notesText, "Notes: notes noted;"));
+    }
+    
+    @Test
+    void doImport_should_parse_stdyDscr_othrStdyMat_related_metadata_fields() throws XMLStreamException, ImportException, IOException {
+        // given
+        String ddiXml = UnitTestUtils.readFileToString("xml/export/ddi/dataset-all-ddi-metadata-fields.xml");
+        
+        // when
+        DatasetDTO datasetDto = importDdiService.doImport(ImportType.HARVEST, ddiXml);
+        
+        // then
+        DatasetVersionDTO datasetVersionDto = datasetDto.getDatasetVersion();
+        MetadataBlockDTO citationBlock = datasetVersionDto.getMetadataBlocks().get("citation");
+        
+        // TODO it is compound field
```

Comment:
This (and the next TODO) is a note for a future or a leftover?

Reply:
Actually we will have to correct that. We would have to either change the field to relatedmaterialcitation or make parsing analogous to what is already done for related publication. I will make a task for that